### PR TITLE
NullSafety, ElevatedButton

### PR DESCRIPTION
**Pull request by joaovvrodrigues**
* Added compatibility with NullSafety
* Removed RaisedButton deprecated
* Add new ElevatedButton
* Update dependencies
* Added Pedantic recommendations, analyzer settings and best practices used internally at Google.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,20 +1,21 @@
 name: rounded_loading_button
 description: A simple implementation of an animated loading button widget for Flutter
-version: 1.0.18
+version: 2.0.1
```
Comment (chrisedg87) [inline pubspec.yaml:3]: I think this should probably be 2.0.0 to match the changelog.

**Comment by mayurdhaka-suryasoftware** [comment]
Hey @chrisedg87 Any plans on merging this if the comment you left is resolved?

**Review by chrisedg87 — APPROVED** [review]

**Comment by chrisedg87** [comment]
Apologies this one has taken so long to approve. Thanks everyone!

**Comment by chrisedg87** [comment]
Resolves issue #28

**Pull request merged**